src/coreclr/src/hosts/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 include_directories(inc)
 
+add_definitions(-D_FILE_OFFSET_BITS=64)
```

Comment:
This way, it would apply to both Windows and Unix. We don't need it for Windows. Can you please move it to where I've suggested (right before the `add_subdirectory(unixcoreruncommon)`) ?

Reply:
Thanks for your patience. Why is `add_subdirectory(unixcoreruncommon)` under the same `CLR_CMAKE_HOST_WIN32` condition?

Reply:
It is not under the same condition, it is just the way cmake condition parameters are written. When you have a condition in `if`, you just repeat the same one in `else` and `endif`. In those two, it serves just as a comment to indicate which `if` these are related to. And cmake actually verifies that it matches, so it is a bit more than a comment.